syncd/CommandLineOptionsParser.cpp

```diff
@@ -164,6 +172,8 @@ void CommandLineOptionsParser::printUsage()
     std::cout << "        Enable consisteny check DB vs ASIC after comparison logic" << std::endl;
     std::cout << "    -s --syncMode" << std::endl;
     std::cout << "        Enable synchronous mode" << std::endl;
```

Comment:
i already added this in log warn message, do you want me to update it here as well?

Reply:
I thought the warn message could only be seen in the log after running the command. When someone uses this command, he/she might read this help message. So I thought adding a message here might be useful to prevent people from using this option in the first place?

Reply:
updated